cob_moveit_config/bin/add_robot.sh

```diff
@@ -16,5 +16,5 @@ moveit_setup_assistant_config:
     package: cob_hardware_config
     relative_path: robots/ROBOT/urdf/ROBOT.urdf.xacro
   SRDF:
-    relative_path: moveit/config/ROBOT.srdf
```

Comment:
`relative_path: moveit/config/ROBOT.srdf` is correct see https://github.com/ipa320/cob_robots/tree/indigo_dev/cob_moveit_config/robots/cob4-5/moveit/config

please revert